src/factories/request.js

```diff
@@ -73,12 +57,14 @@ class RequestFactory {
       : createAuthRequest(config)
 
     promise
-      .then(response => {
-        const credentials = new Credentials(
```

Comment:
Thanks for the cleanup Sam! I didn't realize this was the only usage of Credentials() too.

Reply:
@shaunmaharaj seems a bit verbose so I stripped it out, unless you have any objections

Reply:
@samblacklock nope, I agree. Go for it!